shared/chat/conversation/info-panel/notifications/index.js

```diff
@@ -18,90 +18,101 @@ export type Props = {
   toggleChannelWide: () => void,
 }
 
-const UnmutedNotificationPrefs = (props: Props) => (
-  <React.Fragment>
-    <Checkbox
-      checked={!props.channelWide}
-      label=""
-      labelComponent={
-        <Text type="Body">
-          Ignore <Text type="BodySemibold">@here</Text> and <Text type="BodySemibold">@channel</Text> mentions
-        </Text>
-      }
-      onCheck={props.toggleChannelWide}
-    />
-
-    <Box style={isMobile ? styleHeaderMobile : styleHeader}>
-      <Text type="BodySmallSemibold">Desktop notifications</Text>
-      <Icon
-        style={{paddingLeft: globalMargins.xtiny}}
-        fontSize={isMobile ? 20 : 16}
-        color={globalColors.black_20}
-        type="iconfont-notifications-desktop"
+const UnmutedNotificationPrefs = (props: Props) => {
+  const fullyEnabled = props.desktop === 'onAnyActivity' && props.mobile === 'onAnyActivity'
```

Comment:
shouldn't this be fullyDisabled

Reply:
I mean the notifications being fully enabled, will come up with a better name